ClinGen/clincoded
# Connects to #1872: Adds link to evidence summary text

**Steps to test #1872**

1. Click **View list of all Gene-Disease Records** and select **DICER1**
2. Click **Classification Matrix** in the gene record header
3. You should expect to see the **View Example Evidence Summary Text** button in the **Gene/Disease Pair** table
4. Click the button and expect a new tab to open with this link: https://clinicalgenome.org/site/assets/files/3439/gene_curation_evidence_summary_12_11_18.pdf

------

I wasn't sure if the link to the PDF belonged in the external_url_map. I saw that other clinicalgenome.org links weren't, so I just followed suit.
